src/WebJobs.Script/Metrics/HostMetricsMeter.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Azure.WebJobs.Script.Metrics
+{
+    public class HostMetricsMeter
```

Comment:
nit:

1. extract interface (so it can be unit tested) _or_ make all methods `virtual`.
2. `Meter` sounds superfluous to me. What do you think of just `HostMetrics` (or `IHostMetrics` if you extract an interface)? Would need to rename `HostMetrics` payload to something else - maybe `HostMetricsPayload`? That type will go away eventually so not concerned about an ugly name.

Reply:
I was also thinking ScaleMetrics - also don't care too much as the plan will be to remove this